client/me/purchases/payment-methods/components/payment-method-edit-dialog.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Dialog } from '@automattic/components';
+import { useTranslate, TranslateResult } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
```

Comment:
Minor, but this could be `import type`:

```suggestion
import { useTranslate, TranslateResult } from 'i18n-calypso';
import type { TranslateResult } from 'i18n-calypso';
import type { FunctionComponent } from 'react';
```

Reply:
Fixed.